python/src/com/jetbrains/python/psi/impl/references/PyQualifiedReference.java

```diff
@@ -158,6 +159,11 @@ private void addImplicitResolveResults(String referencedName, ResolveResultList
       }
     }
 
+    PyClassAttributesIndex.findAttributes(referencedName, project, scope).forEach(classAttribute -> {
```

Comment:
If we add instance attributes here alongside with class-level attributse, we could drop `PyInstanceAttributesIndex` completely.